### DISCUSSION TITLE
Modal & New Tab?

### DISCUSSION THREAD

**Discussion by stuu3k** [Lissy93/dashy#492] [Ideas]
Hi Lissy,

The more I use it the more I am torn between two modes for many of my sites, like for example Portainer which is very easy and quick to restart or make a small change to a docker in the model mode I use it all the time.

However sometimes I like to open it in a new tab, especially if I know I am going to be in and out of it or in it for a while, so I find myself typing part of it and going the old way rather than usnig dashy as if I Middle Mouse on the shortcut rather than opeing the shortcut on in a new tab like expected I just get a new copy of Dashy instead. :(

IDK maybe it's not possible but it would be really nice if you could have Model with the option to New Tab with Middle Mouse.

If it's can't be done that way maybe an way to set an alt-launch mode or something, maybe a small little icon you could click on instead to launch in new tab or an specific area of the icon IDK just thinking aloud.

Anyway thanks again, keep up the great work! 👍

**Comment by lissy93** [comment]
Yeah, that's a good idea, I could make the <kbd>Alt</kbd> + <kbd>Click</kbd> customizable. So that clicking will launch in main target, and Alt + Tab will launch in secondary target :)

In the meantime, you can right-click an item, and that shows all options :)
<img src="https://user-images.githubusercontent.com/1862727/149828399-5ef22418-ce88-4110-8e1e-6b7295758690.png" width="160" />

Update: Oh, I see what you mean about it opening Dashy in a new tab on middle click.... That's a bug, I see where I went wrong- I will fix that this weekend, this is where I went wrong..

https://github.com/Lissy93/dashy/blob/bb4658e16c7bf9be15bd338d06929c0a103d3f80/src/components/LinkItems/Item.vue#L141-L147

**Reply by lissy93** [reply]
Sorry for the delay, this is now implemented in #557 :)
There's also some more improvements planned for better opening items coming up in the next few weeks...

**Comment by stuu3k** [comment]
Ah even better, once again thanks for prompt reply! 🥇
